src/index.js

```diff
@@ -22,4 +22,4 @@ export const flowMap = (...fns) => _.map(_.flow(...fns))
 // Misc
 // ----
 export const testRegex = regex => regex.test.bind(regex)
-export const compareDeep = _.curry((path, item, other) => _.get(path, item) == other)
+export const compareDeep = _.curry((path, item, other) => _.get(path, item) === other)
```

Comment:
just realized this actually does need to be double equal so it doesn't change the behavior of the method